instrumentation/graphql-java/graphql-java-common/library/src/main/java/io/opentelemetry/instrumentation/graphql/internal/GraphqlAttributesExtractor.java

```diff
@@ -19,13 +22,6 @@
  */
 final class GraphqlAttributesExtractor
     implements AttributesExtractor<OpenTelemetryInstrumentationState, ExecutionResult> {
-  // https://github.com/open-telemetry/semantic-conventions/blob/main/docs/database/graphql.md
-  private static final AttributeKey<String> OPERATION_NAME =
-      AttributeKey.stringKey("graphql.operation.name");
-  private static final AttributeKey<String> OPERATION_TYPE =
-      AttributeKey.stringKey("graphql.operation.type");
-  private static final AttributeKey<String> GRAPHQL_DOCUMENT =
-      AttributeKey.stringKey("graphql.document");
```

Comment:
Depending on whether https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/11324 is accepted you may need to put these constants back here.

Reply:
Shall I put these back now as this seems the to be the safer option?